src/main/java/configuration/AssignmentStrategyConfig.java

```diff
@@ -0,0 +1,5 @@
+package configuration;
+
+public enum AssignmentStrategyConfig {
```

Comment:
Can you put this in Config.java instead of dedicated file?

Reply:
Done